code/modules/client/preferences.dm

```diff
@@ -58,6 +58,7 @@ GLOBAL_LIST_EMPTY(chosen_names)
 	var/preferred_map = null
 	var/pda_style = MONO
 	var/pda_color = "#808000"
+	var/xp_text = FALSE
```

Comment:
hot take, this should be a bitflag so its easier to add toggles later on for other balloon alerts. I made a datumized pref system for Noctra a while back if you need something to look at

Reply:
this is a room temperature take m'lord I agree completely